internal/docs/docs-components/navigation.js

```diff
@@ -30,10 +30,16 @@ const Navigation = styled.nav`
     padding: ${spacing.medium} 0;
     color: ${colors.base.grayLightest};
   }
+
   a > span {
     display: inline-block;
     vertical-align: middle;
   }
+
+  a > span + span {
+    margin-left: 8px;
```

Comment:
Can we use tokens for this? https://github.com/auth0/cosmos/tree/master/core/tokens